app/models/subnet.rb

```diff
@@ -213,6 +213,12 @@ def has_vlanid?
     self.vlanid.present?
   end
 
+  # Given a parameter name, return the (first) value, or nil if nothing found
+  def parameter_for_name(name)
```

Comment:
@dLobatog is it possible to add a `subnet_param` to `lib/foreman/renderer.rb` ? I see eg. `ALLOWED_HOST_HELPERS`, is this relevant too? Should I add `subnet_param` to that array, or should there be an different implementation here? I don't see it done easily (but I have limited experience with this)